### ISSUE TITLE
New release version with `existing_ids`

### ISSUE THREAD

**Issue by lkursell** [johnchase/cual-id#40]

**Comment by c42f** [comment]
+1

It would be great to have a new version available via conda. Without `existing-ids` I'll have to persuade my colleagues to install the development version which may put it in the too hard basket ;-)

**Comment by johnchase** [comment]
@c42f I agree this is a bit overdue. I will see if I can get this together soon
